ACE/ace/Basic_Types.h

```diff
@@ -70,6 +70,12 @@
 #  include <inttypes.h>
 # endif
 
+# if defined (ACE_HAS_CPP11)
+#  include <cstdint>
```

Comment:
This also is included a could of lines up

Reply:
I could merge them into one.